jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -117,6 +117,15 @@
                 export KUBE_AUTOSCALER_MAX_NODES=5
                 export KUBE_AUTOSCALER_ENABLE_SCALE_DOWN=true
                 export KUBE_ADMISSION_CONTROL="NamespaceLifecycle,InitialResources,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota"
+        - 'gce-alpha-features': # kubernetes-e2e-gce-alpha-features
+            description: 'Run alpha feature E2E tests on GCE.'
+            timeout: 180
+            job-env: |
+              export KUBE_FEATURE_GATES="AllAlpha=true"
```

Comment:
AllAlpha

On Mon, Aug 22, 2016 at 8:37 PM, Michael Taufen <notifications@github.com>
wrote:

> In jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml
> <https://github.com/kubernetes/test-infra/pull/427#discussion_r75798365>:
>
> > @@ -117,6 +117,15 @@
> >                  export KUBE_AUTOSCALER_MAX_NODES=5
> >                  export KUBE_AUTOSCALER_ENABLE_SCALE_DOWN=true
> >                  export KUBE_ADMISSION_CONTROL="NamespaceLifecycle,InitialResources,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota"
> > +        - 'gce-alpha-features': # kubernetes-e2e-gce-alpha-features
> > +            description: 'Run alpha feature E2E tests on GCE.'
> > +            timeout: 180
> > +            job-env: |
> > +              export KUBE_FEATURE_GATES="AllAlpha=true"
>
> Is it supposed to be AllAlpha or allAlpha here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/427/files/91e99c6172a4ffdde370c0423d94b0f2828b685b#r75798365>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVFhF1enlFfw4Tqy3g-2t2iFoszL_ks5qimr_gaJpZM4JqTfe>
> .
>